site/src/playroom.components.ts

```diff
@@ -1,3 +1,3 @@
 import 'braid-src/entries/reset';
```

Comment:
Is there a reason to not use the public reset too?

Reply:
Yes, but the explanation is long and complicated. @askoufis has the results from the Playroom dependency analysis. TLDR: Playroom needs changes to ensure the same reset (not ESM and CJS) is imported only once per entry point.

Reply:
Thanks sir 🙏